powerauth-nextstep-client/src/main/java/io/getlime/security/powerauth/lib/nextstep/client/NextStepClient.java

```diff
@@ -91,28 +91,31 @@ private RestTemplate defaultTemplate() {
      * Calls the operation endpoint via POST method to create a new operation.
      *
      * @param operationName operation name
+     * @param operationId operation ID (optional)
```

Comment:
Lazy fix. :) If the parameter is optional, create a method that does not have it - as an API consumer, how do I know what "(optional)" means here?